src/hotspot/share/runtime/os.cpp

```diff
@@ -2333,7 +2333,7 @@ char* os::reserve_memory_special(size_t size, size_t alignment, size_t page_size
   char* result = pd_reserve_memory_special(size, alignment, page_size, addr, executable);
   if (result != nullptr) {
     // The memory is committed
-    MemTracker::record_virtual_memory_reserve_and_commit((address)result, size, CALLER_PC);
+    MemTracker::record_virtual_memory_reserve_and_commit((address)result, size, CALLER_PC, mtNone);
```

Comment:
reserve_memory_special should take a mem tag, but I guess you intend to do that as a follow-up RFE?

Reply:
Yes, I'm trying to keep changes small and not overdo it. There will be follow-ups.